File: Assignment No.33/Program3.java
/*Problem Statement : Write java program which accept N Number from user and Display the all elemnets which are even and divisible by 5 .
Input : N - 6
		Elements : 85 66 3 80 93 88
Output : 80 
*/
import java.util.*;

class ArrayDemo
{
	public void Diffrence(int Arr[])
	{
		int iCnt=0;
		for(iCnt=0;iCnt<Arr.length;iCnt++)
		{
			if(((Arr[iCnt] % 5)==0)&&((Arr[iCnt] % 2)==0))
			{
				System.out.println("Elements divisible by 5 and even Number :"+Arr[iCnt]); 
			}
		
		}
		
	}
}
class Program3
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		ArrayDemo aobj=new ArrayDemo();

		int iSize=0,iCnt=0,iRet=0;
		System.out.println("Enter the Number:");
		iSize=sobj.nextInt();

		int Arr[]= new int[iSize];

		System.out.println("Enter the elements :"+ iCnt+1);
		for(iCnt=0;iCnt < iSize;iCnt++)
		{
			Arr[iCnt]=sobj.nextInt();
		}
		aobj.Diffrence(Arr);
	}
}

File: Assignment No 6/Program3.c
//Write a program which accept number from user and print its number line.
//Input : 4
//Output : -4 -3 -2 -1 0 1 2 3 4

#include<stdio.h>
void Display(int iNo)
{
	if(iNo < 0)
	{
		iNo=-iNo;
	}
	int i=0;

	for(i=-iNo;i<=iNo;i++)
	{
		printf("%d\t",i);
	}
}
int main()
{
	int iValue=0;
	printf("Enter the number :\n");
	scanf("%d",&iValue);
	Display(iValue);
	return 0;
}